Ambit/Source/Ambit/Mode/AmbitDetailCustomization.cpp

```diff
@@ -77,472 +77,473 @@ void FAmbitDetailCustomization::CustomizeDetails(
     // Build Scenario Settings Menu
     // The order that you add properties here is the order they actually appear
     IDetailCategoryBuilder& ScenarioSettingsCategory = DetailBuilder.
-        EditCategory("Scenario Settings");
+            EditCategory("Scenario Settings");
 
     // Format Scenario Name and Import Button
     const FString KHintTextScenarioName = "Import Scenario Definition File or Type Name for New Scenario";
     TSharedRef<IPropertyHandle> PropertyHandle_ScenarioName = DetailBuilder.
-        GetProperty(GET_MEMBER_NAME_CHECKED(UAmbitObject, ScenarioName));
+            GetProperty(GET_MEMBER_NAME_CHECKED(UAmbitObject, ScenarioName));
     ScenarioSettingsCategory.AddProperty(PropertyHandle_ScenarioName)
-        .CustomWidget()
-        .NameContent()
-        [
-            PropertyHandle_ScenarioName->CreatePropertyNameWidget()
-        ]
-        .ValueContent()
-        .MinDesiredWidth(200.0f)
-        [
-            SNew(SHorizontalBox)
-            + SHorizontalBox::Slot()
-            .FillWidth(1)
+                            .CustomWidget()
+                            .NameContent()
```

Comment:
We avoid to use ReSharper on the Slate in the past and it seems the ReSharper still doesn't work well with it.

Reply:
This is a bit vague. Can you help me understand exactly what you experienced in the past so I can verify if there is still an issue?

Reply:
Will revert Slate changes, and add Formatter ignore around.